tests/vehicle_details.yaml

```
@@ -0,0 +1,89 @@
+appId: com.anonymous.PublicMobileApp
+---
+# ----------------------------------- VEHICLE LIST TEST -------------------------------------------------------------------
+- tapOn: "Vozila"
+- waitForAnimationToEnd
+
+# verify data is visible 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)